src/preferences/dialog/dlgprefcolors.cpp

```diff
@@ -84,31 +90,39 @@ DlgPrefColors::~DlgPrefColors() {
 void DlgPrefColors::slotUpdate() {
     comboBoxHotcueColors->clear();
     comboBoxTrackColors->clear();
+    comboBoxKeyColors->clear();
     checkboxKeyColorsEnabled->setChecked(
             m_pConfig->getValue(kKeyColorsEnabledConfigKey,
                     BaseTrackTableModel::kKeyColorsEnabledDefault));
     for (const auto& palette : std::as_const(mixxx::PredefinedColorPalettes::kPalettes)) {
-        QString paletteName = palette.getName();
+        const char* paletteName = qPrintable(palette.getName());
```

Comment:
The returned pointer becomes invalid immediately after the call. If you like to reuse the value, store the translated string in a QString()